app/browser/reducers/tabsReducer.js

```diff
@@ -158,6 +119,7 @@ const tabsReducer = (state, action, immutableAction) => {
       break
     case appConstants.APP_TAB_UPDATED:
       state = tabState.maybeCreateTab(state, action)
+      // tabs.debugTabs(state)
```

Comment:
comment left in here :)

Reply:
that's on purpose because I toggle it often